client/js/models/archive.js

```diff
@@ -79,7 +80,6 @@ module.exports = {
   effects: {
     new: function (data, state, send, done) {
       drive = getDrive()
-      var _fs = {}
```

Comment:
Note that in the client if you load a new dat url, we call the effect again, so _fs should be per-archive